source/simulator/parameters.cc

```diff
@@ -884,13 +884,19 @@ namespace aspect
 
       prm.enter_subsection ("Stabilization parameters");
       {
+        prm.declare_entry ("Stabilization method", "entropy viscosity",
+                           Patterns::Selection("entropy viscosity|SUPG"),
+                           "Select the method for stabilizing the advection equation. The original "
+                           "method implemented is 'entropy viscosity' as described in \\cite {KGB12}.");
```

Comment:
```suggestion
                           "method implemented is 'entropy viscosity' as described in \\cite {KHB12}.");
```

Also add a sentence that says that SUPG is currently experimental.